package.json

```diff
@@ -26,7 +26,7 @@
     "@angular/platform-browser": "~10.0.11",
     "@angular/platform-browser-dynamic": "~10.0.11",
     "@angular/router": "~10.0.11",
-    "@stoplight/elements-web-components": "beta",
+    "@stoplight/elements-web-components": "6.0.0-beta.163",
```

Comment:
Please make this a "^"

Reply:
@marcelltoth Wouldn't that update to the latest version, which is what I wanted to avoid here?

Reply:
oooooh, my bad